torch/nn/quantizable/modules/activation.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, embed_dim: int, num_heads: int,
         # TODO: The use of the `_LinearWithBias` increases the quantization noise
```

Comment:
is this comment still relevant?

Reply:
I went back and forth on this, but I left it in for the case where an older model is loaded and then quantized. Is this valid? The comment should probably be at least updated to reflect that `out_proj` could or could not be a `_LinearWithBias`.